src/WebJobs.Script.WebHost/Security/Authorization/FunctionAuthorizationRequirement.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Security.Authorization
+{
+    public class FunctionAuthorizationRequirement : IAuthorizationRequirement
+    {
```

Comment:
I don't understand this empty class - the interface is even empty  - just a marker?

Reply:
Yes, `IAuthorizationRequirement` is a marker interface. The requirement type defines the requirement that must be handled/met and carries any all the relevant data (you can see another example in the `AuthLevelRequirement`). The `FunctionAuthorizationRequirement` just doesn't need any additional information (yet).